include/tilck/common/string_util.h

```diff
@@ -29,14 +29,28 @@
 
 #endif
 
-int strcmp(const char *s1, const char *s2);
-int strncmp(const char *s1, const char *s2, size_t n);
-int memcmp(const void *m1, const void *m2, size_t n);
-char *strstr(const char *haystack, const char *needle);
-char *strcpy(char *dest, const char *src);
-char *strncpy(char *dest, const char *src, size_t n);
-char *strcat(char *dest, const char *src);
-char *strncat(char *dest, const char *src, size_t n);
+#if !defined(TESTING) && !defined(USERMODE_APP)
+   int strcmp(const char *s1, const char *s2);
+   int strncmp(const char *s1, const char *s2, size_t n);
+   int memcmp(const void *m1, const void *m2, size_t n);
+   char *strstr(const char *haystack, const char *needle);
+   char *strcpy(char *dest, const char *src);
+   char *strncpy(char *dest, const char *src, size_t n);
+   char *strcat(char *dest, const char *src);
+   char *strncat(char *dest, const char *src, size_t n);
+
+   int isxdigit(int c);
+   int isspace(int c);
+#endif
+
+int tilck_strcmp(const char *s1, const char *s2);
```

Comment:
These were supposed to be **outside** of the `#if !defined(TESTING) && !defined(USERMODE_APP)` block, right? It ends a few lines below. There is also a comment.